airflow/jobs/scheduler_job.py

```diff
@@ -684,7 +684,7 @@ def _execute(self) -> None:
 
             settings.Session.remove()  # type: ignore
         except Exception:
-            self.log.exception("Exception when executing SchedulerJob._run_scheduler_loop")
+            self.log.error("Exception when executing SchedulerJob._run_scheduler_loop")
```

Comment:
This does not change the logging level, only suppress the exception from being logged. I don’t think we should do this here.

Reply:
The re-raising exception below will not be caught by any upper caller of _execute(), the exception-level logging here leads to duplicated exception stack printing

Reply:
> duplicated exception stack printing

Where is the second time the exception is printed? If it is not caught (and the exception is printed by Python on exit), these are not conceptually duplicates because not all logs are handled by the console, and removing the log here makes the exception invisible if the outmost exception is not connected to a console (and the one here is logged to a file).

Reply:
Yep. I checked and there does not sem to be any caller - and still we want to log thiss one to scheduler logger.